docs/get-started/quick-start.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Quick start"
+layout: default
+sort: 2
+---
+
+# Quick start
+
+Get the source code
+
+```bash
+git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator
+```
+
+Deploy the operator
+
+```bash
+IMAGE_TAG=k8s.gcr.io/nfd/node-feature-discovery-operator:v0.2.0
```

Comment:
I'd suggest to parameterize the operator version in `docs/_config.yml`. Makes future updates of the version less error-prone as there's only one well-defined place where it needs to be changed.